downgrade

```diff
@@ -209,7 +239,13 @@ main() {
     # shellcheck disable=SC2207
     installed=($(previously_installed "$term"))
     current=$(currently_installed "$term")
-    process_term "$term" || exit_code=1
+    if ! process_term "$term"; then
+      {
+        gettext "Unable to downgrade $term"
```

Comment:
I think you have to do something here so `$term` doesn't end up in the translation key and never matching. Is this what `eval_gettext` is for?

Reply:
Ah yes true, that is exactly how `eval_gettext` was used for another option.